pylearn2/expr/basic.py

```diff
@@ -242,23 +242,40 @@ def is_binary(x):
     return np.all( (x == 0) + (x == 1))
 
 
-def log_sum_exp(log_A, axis=None):
+def log_sum_exp(A=None, axis=None, log_A=None):
     """
     A numerically stable expression for
-    `T.log(T.exp(log_A).sum(axis=axis))`
+    `T.log(T.exp(A).sum(axis=axis))`
 
     Parameters
     ----------
-    log_A : tensor_like
-        Log of tensor A
+    A : theano.gof.Variable
+        A tensor we want to compute the log sum exp of
     axis : int, optional
         Axis along which to sum
+    log_A : deprecated
+        This function used to be specified in terms of log(A),
+        which was unnecessary and confusing. Just use A.
```

Comment:
I would be more clear like:

The old parameter name for A was log_A. This was confusing as it wasn't defined in term of log_A. We just renamed the parameter name to A. The behavior is the same.